x/evm/keeper/msg_server.go

```diff
@@ -776,10 +776,13 @@ func (s msgServer) VoteConfirmDeposit(c context.Context, req *types.VoteConfirmD
 		return nil, fmt.Errorf("cross-chain sender has no recipient")
 	}
 
+	height, _ := keeper.GetRequiredConfirmationHeight(ctx)
```

Comment:
Don't think you should ignore the error here.

Reply:
Thanks - have handled the bool that's returned